path_finding.asciidoc

```diff
@@ -122,3 +127,21 @@ These three activities can be repeated in a _payment round_ if we use the failur
 In the next sections we will look at each of these steps in more detail, as well as more advanced payment strategies.
 
 === Channel graph construction
+
+In <<gossip>> we explain the three main messages that nodes "gossip": +node_announcement+, +channel_announcement+, and +channel_update+. These three messages allow any node to gradually construct a "map" of the Lightning Network in the form of a _channel graph_. Each of these messages provides a critical piece of information for the channel graph:
+
+node_announcement:: This contains the information about a node on the Lightning Network, such as its node ID (public key), network address (e.g. IPv4/6 or Tor), capabilities/features etc.
+
+channel_announcement:: This contains the capacity and channel ID of a public (announced) channel between two nodes and proof of the channel's existence and ownership.
```

Comment:
I assume this is already incorperating gossip? In which case I would explain how the short channel id has the standardized format `height`x`index`x`output`